pkg/mbedtls/include/riot_config.h

```diff
@@ -57,20 +59,24 @@
  *            digests and ciphers instead.
  *
  */
+#if IS_ACTIVE(CONFIG_MBEDTLS_SHA256_ALT)
 #define MBEDTLS_SHA256_ALT 1
+#endif
```

Comment:
I think this may cause confusion. If the exposed configuration parameters to the user are the `CONFIG_` ones, then that's what should be documented in Doxygen. So in this case the user should find `CONFIG_MBEDTLS_SHA256_ALT` instead of `MBEDTLS_SHA256_ALT`.

Additionally, as all the configurations seem to default to `1`, we may need to guard them around a check for `CONFIG_KCONFIG_USEMODULE_MBEDTLS`, because `false` in Kconfig translates to a not defined macro.